src/sinol_make/commands/run/__init__.py

```diff
@@ -706,6 +706,42 @@ def validate_arguments(self, args):
 
         return compilers, timetool_path
 
+    def set_scores(self):
+        self.tests = self.get_tests(self.args.tests)
+        self.groups = list(sorted(set([self.get_group(test) for test in self.tests])))
+        self.scores = collections.defaultdict(int)
+
+        if 'scores' not in self.config.keys():
+            print(util.warning('Scores are not defined in config.yml. Points will be assigned equally to all groups.'))
+            num_groups = len(self.groups)
+            self.scores = {}
+            if self.groups[0] == 0:
+                num_groups -= 1
+                self.scores[0] = 0
+
+            points_per_group = 100 // num_groups
+            for group in self.groups:
+                if group == 0:
+                    continue
+                self.scores[group] = points_per_group
+
+            if points_per_group * num_groups != 100:
+                self.scores[self.groups[-1]] += 100 - points_per_group * num_groups
+
+            print("Points will be assigned as follows:")
+            total_score = 0
+            for group in self.scores:
+                print("%2d: %3d" % (group, self.scores[group]))
+                total_score += self.scores[group]
+            if total_score != 100:
+                print(util.warning("WARN: Scores sum up to %d instead of 100." % total_score))
```

Comment:
This warning isn't reachable, right? It should be printed when the user's own configuration is wrong, not the one provided by us.

Reply:
Yes, I changed that